tzdata/supplemental/timezoneTypes.txt

```diff
@@ -80,7 +80,10 @@ timezoneTypes:table(nofallback){
             "America:Thunder_Bay"{"America/Toronto"}
             "America:Virgin"{"America/St_Thomas"}
             "America:Yellowknife"{"America/Edmonton"}
-            "Antarctica:South_Pole"{"Pacific/Auckland"}
+            "Antarctica:South_Pole"{
```

Comment:
I think this is bad, did you pull in https://github.com/unicode-org/cldr/pull/4592?

Reply:
#4592 was merged 12 hours ago, so nope. Let me sync my CLDR checkout.

Thank you for pointing that out.